src/nanoemoji/write_font.py

```diff
@@ -363,6 +363,17 @@ def _draw_glyph_extents(
     return glyph
 
 
+def _draw_notdef(config: FontConfig, ufo: ufoLib2.Font):
+    glyph = ufo[".notdef"]
+    pen = glyph.getPen()
+    pen.moveTo((0, 0))
+    pen.lineTo((config.width, 0))
+    pen.lineTo((config.width, config.upem))
+    pen.lineTo((0, config.upem))
+    pen.endPath()
+    glyph.width = config.width
```

Comment:
FWIW, a more canonical way to draw a .notdef glyph (i.e. stroked rectangle centered within the advance width that vertically fits ascender/descender) can be found in ufo2ft outlineCompiler:
https://github.com/googlefonts/ufo2ft/blob/main/Lib/ufo2ft/outlineCompiler.py#L1552-L1580

maybe you could borrow that one? You could even import the ufo2ft `StubGlyph` object and pass it to ufo2ft's `compile*` methods' [`notdefGlyph` parameter](https://github.com/googlefonts/ufo2ft/blob/f4f49e444bf7a1c1b02f1a98591807cb69d4015b/Lib/ufo2ft/__init__.py#L91).

Reply:
Nice. Note to self: StubGlyph, when name=".notdef", will draw a nice notdef glyph

Reply:
Further note to self: do this always when generating the UFO

Reply:
Done, ty